### PR TITLE
standardrb: init at 1.35.1

### PR DESCRIPTION
**Pull request by 3JlOy-PYCCKUi**
## Description of changes

[standard](https://github.com/standardrb/standard) gem is used for linting/formatting ruby code

there is a pr (215167) for this gem, but it has merge conflicts and it looks like the author is not going to fix it

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/top-level/ruby-packages.nix [hunk 1]
```diff
@@ -1151,21 +1151,21 @@
...
     groups = ["default"];
     platforms = [];
     source = {
       remotes = ["https://rubygems.org"];
-      sha256 = "0w3xhay1z7qx9ab04wmy5p4f1fadvqa6239kib256wsiyvcj595h";
+      sha256 = "1wr5gxnr158scdp56zj1x5q47iaxbpm84nc75vyi6nqzyl4bmln4";
       type = "gem";
     };
-    version = "1.19.1";
+    version = "2.0.0";
```
Comment (SuperSandro2000) [inline pkgs/top-level/ruby-packages.nix:1168]: We mostly want to stay on 1.X.X for now

File: pkgs/top-level/ruby-packages.nix [hunk 2]
```diff
@@ -1300,10 +1300,10 @@
     platforms = [];
     source = {
       remotes = ["https://rubygems.org"];
-      sha256 = "0ix9rp9rsrwin38z2aqiwpxc8p2dzl6m3ra47az47fsija1cb2qf";
+      sha256 = "1xa70k9g76q0087wcdiy4gqflsfzh8dkzv2agk1sdlqcj54yhbmd";
       type = "gem";
     };
-    version = "2.0.0";
+    version = "3.0.0";
```
Comment (SuperSandro2000) [inline pkgs/top-level/ruby-packages.nix:1306]: same here

**Review by SuperSandro2000 — COMMENTED** [review]
if we want to merge this directly to master, we must remove the other updates or target staging

---

This PR rebuilds a lot of packages which means we must target staging. Please follow the [contributing guide](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#rebasing-between-branches-ie-from-master-to-staging) to not potentially ping a lot of people.

**Comment by jhult** [comment]
@3JlOy-PYCCKUi, any updates on this?

**Comment by jhult** [comment]
Any news on this?

**Comment by kimat** [comment]
> there is a pr (215167) for this gem, but it has merge conflicts and it looks like the author is not going to fix it

@3JlOy-PYCCKUi I fixed the merge conflict in #215167 now.
Note that one doesn't get notified when the "merge conflict" label is added to one of their PRs.
Also, the PR author(s) don't get notified if you omit the `#` when mentioning a PR.

 > if we want to merge this directly to master, we must remove the other updates 
> We mostly want to stay on 1.X.X for now

@SuperSandro2000 these remarks have been addressed in the original PR #215167 (which was created more than one year prior to this very similar PR).

**Pull request closed without merging**